openapi-generator/python_lang.yaml

```diff
@@ -3,7 +3,10 @@ generatorName: python
 outputDir: clients/python
 packageName: phrase_api
 projectName: phrase-api
-packageVersion: 1.9.0
```

Comment:
This should be managed by release-please and is moved to `version.txt` in python template directory.